index.html

```diff
@@ -1424,7 +1425,8 @@ <h2 id="objects">Object Functions</h2>
       <p id="pairs">
         <b class="header">pairs</b><code>_.pairs(object)</code>
         <br />
-        Convert an object into a list of <tt>[key, value]</tt> pairs.
+        Convert an object into a list of <tt>[key, value]</tt> pairs. Inverse of first
+        usage of <code>_.object</code>. 
```

Comment:
> Inverse of first usage of <code>_.object</code>.

In order to better fit with the rest of our documentation, could you reword this as a complete sentence?